### PR TITLE
Updated Agama specification

### PR DESCRIPTION
**Pull request by tbazant**

### CODE REVIEW

File: references/alp-installer-profile-based-automated-deployment-options.xml [hunk 1]
```diff
@@ -83,11 +131,27 @@
       </listitem>
     </varlistentry>
     <varlistentry>
-      <term>storage.devices</term>
+      <term>storage.bootDevice</term>
       <listitem>
         <para>
-          Array of devices where &productnameshort; will be installed. For
-          example, <literal>["/dev/sda"]</literal>.
+          "Device used for booting, for example, <filename>/dev/sda</filename>. By default, all file
```
Comment (dariavladykina) [inline references/alp-installer-profile-based-automated-deployment-options.xml:137]:
```suggestion
          Device used for booting, for example, <filename>/dev/sda</filename>. By default, all file
```

**Review by dariavladykina — COMMENTED** [review]
LGTM, just a typo. Thanks!

File: references/alp-installer-profile-based-automated-deployment-options.xml [hunk 2]
```diff
@@ -42,19 +42,51 @@
...
     <varlistentry>
-      <term>software.product</term>
+      <term>software.patterns</term>
       <listitem>
         <para>
-          Mandatory product identifier. For example, ALP-Dolomite.
+          List of software patterns to install.
         </para>
       </listitem>
     </varlistentry>
```
Comment (jreidinger) [inline references/alp-installer-profile-based-automated-deployment-options.xml:92]: This part looks good for me. For localization I will keep it for @mvidner

File: references/alp-installer-profile-based-automated-deployment-options.xml [hunk 3]
```diff
@@ -74,6 +106,22 @@
...
+    <varlistentry>
+      <term>localization.keyboard</term>
+      <listitem>
+        <para>
+          Keyboard layout
```
Comment (mvidner) [inline references/alp-installer-profile-based-automated-deployment-options.xml:113]:
```suggestion
          Keyboard layout, for example, <literal>cz(qwerty)</literal>.
```

File: concepts/alp-installer-profile-based-automated-deployment.xml [hunk 4]
```diff
@@ -43,11 +43,7 @@
     "product": "ALP-Bedrock"
```
Comment (mvidner) [inline concepts/alp-installer-profile-based-automated-deployment.xml:43]: Please change `"en_US"` to `"en_US.UTF-8"` above

**Review by tbazant — COMMENTED** [review]
applied suggestions

**Pull request merged**